src/main/java/io/seqera/tower/cli/commands/AbstractApiCmd.java

```diff
@@ -260,7 +261,7 @@ protected String workspaceRef(Long workspaceId) throws ApiException {
     public Integer call() {
         try {
             Response response = exec();
-            if (app().json) {
+            if (app().output != null && Objects.equals(app().output, OutputType.json)) {
```

Comment:
I think this can be simplify to only `app().output == OutputType.json`

Reply:
It was a fancy Intellij suggestion... it pisses me off when the IDE underlines the code in orange.. will revert :)